.github/workflows/nodepackage.yml

```diff
@@ -45,11 +45,17 @@ jobs:
             export CC="g++-4.8"
           fi
           npm install
+
+          if [ ${{ matrix.node }} = 8 ]; then
+            # mocha 8+ is not compatible with Node 8
+            npm install mocha@7.2.0
+          fi
+
           npm run test
       - name: Lint
         run: |
-          if [ ${{ matrix.node }} = 6 ]; then
-            # ESLint 6+ is not compatible with Node 6
+          if [ ${{ matrix.node }} = 8 ]; then
```

Comment:
keep node6 lines and add elif I will merge it

Reply:
Even if node 6 is not on github actions ci file?

Reply:
it will preserve the legacy info and the comment wont be lost
but it's a not a big deal

Reply:
Done